lisa/messages.py

```diff
@@ -177,6 +181,15 @@ class NetworkUDPPerformanceMessage(PerfMessage):
     packet_size_kbytes: Decimal = Decimal(0)
 
 
+@dataclass
+class LTPResultMessage(PerfMessage):
```

Comment:
It doesn't need to create a message type for each community test. It's not easy to maintain. We can use a field like `tool` to identify which community test it is.

Reply:
If it's special field, it can be put into the information as dict. For below ones, see each comment.

```python
    ltp_version: str = "" # it can be a test_version field in community test message.
    architecture: str = "" # it can be added in the information of base test result message.
    ltp_test_name: str = "" # it's the name field of base test result message.
    ltp_test_result: str = "" # it's the status field of base test result message.
    ltp_test_exit_value: int = 0 # if it's the way to return error message of ltp test case, put it in the message field.
```